README.md

```diff
@@ -70,8 +70,16 @@ python3 -m examples examples/trivial_socket.yaml
 
 #### Running the tests and linters
 
-You can run the linters via the convenience script `tools/lint.py`.
-
-You can run the test framework using the convenience script `tools/test.py` (via `python -m tools.test`). This outputs html details of code coverage in `.coverage_html`.
-
-> :warning: However, as of this commit, no tests exist.
\ No newline at end of file
+You can run the linters via the convenience script `tools/lint.py` (or, more
+correctly, `python3 -m tools.lint`).
```

Comment:
If we're saying "more correctly, thing" should the more correct option be as the suggestion?
```suggestion
You can run the linters via the convenience script `python3 -m tools.lint`
(or `tools/lint.py`).
```

Reply:
Merging this thought with https://github.com/mewler/mewbot/pull/79#discussion_r956585798
I wrote that import trick to make it so the direct exec method worked, as that is the least surprising, but honestly having two subtly different ways to run the code isn't great.